onnxruntime/contrib_ops/cuda/bert/group_query_attention.cc

```diff
@@ -63,6 +63,7 @@ GroupQueryAttention<T>::GroupQueryAttention(const OpKernelInfo& info)
 
   if (!disable_flash_attention_) {
     zeros_ = this->GetScratchBuffer<int>(kZerosCount, nullptr);
+    CUDA_CALL_THROW(cudaMemset(zeros_.get(), 0, kZerosCount * sizeof(int)));
```

Comment:
can this by `cudaMemsetAsync`?

Reply:
We have no stream here, it's in the constructor of the op. (same reason the GetScratchBuffer has to use nullptr vs context->GetCurrentStream())